packages/zowe-explorer/src/utils/ProfileManagement.ts

```diff
@@ -264,44 +297,8 @@ export class ProfileManagement {
         await vscode.commands.executeCommand("zowe.ds.deleteProfile", node);
     }
 
-    private static getPromptChangeForAllTreesOptions(): vscode.QuickPickItem[] {
-        const qpItemAll: vscode.QuickPickItem = {
-            label: localize("ProfileManagement.getPromptChangeForAllTreesOptions.allLbl", "Yes"),
-            description: localize("ProfileManagement.getPromptChangeForAllTreesOptions.allDesc", "Apply to all trees"),
-        };
-        const qpItemCurrent: vscode.QuickPickItem = {
-            label: localize("ProfileManagement.getPromptChangeForAllTreesOptions.currentLbl", "No"),
-            description: localize("ProfileManagement.getPromptChangeForAllTreesOptions.currentDesc", "Apply to current tree selected"),
-        };
-        return [qpItemAll, qpItemCurrent];
-    }
-
-    private static async promptChangeForAllTrees(node: IZoweTreeNode): Promise<vscode.QuickPickItem> {
-        const [qpItemAll, qpItemCurrent] = this.getPromptChangeForAllTreesOptions();
-        if (!TreeProviders.sessionIsPresentInOtherTrees(node.getLabel().toString())) {
-            return qpItemCurrent;
-        }
-        const qp = Gui.createQuickPick();
-        qp.placeholder = localize("ProfileManagement.promptChangeForAllTrees.howToChange", "Do you wish to apply this for all trees?");
-        qp.items = [qpItemAll, qpItemCurrent];
-        qp.activeItems = [qp.items[0]];
-        qp.show();
-        const selection = await Gui.resolveQuickPick(qp);
-        qp.hide();
-        return selection;
-    }
-
-    private static async handleChangeForAllTrees(node: IZoweTreeNode): Promise<boolean> {
-        const selection = await this.promptChangeForAllTrees(node);
-        if (!selection) {
-            return;
-        }
-        const [all] = this.getPromptChangeForAllTreesOptions();
-        return selection.label === all.label;
-    }
-
     private static async handleHideProfiles(node: IZoweTreeNode): Promise<void> {
-        const shouldHideFromAllTrees = await this.handleChangeForAllTrees(node);
+        const shouldHideFromAllTrees = await this.handleChangeForAllTrees(node.getLabel().toString(), false);
```

Comment:
Curious if we can default the `checkPresence` boolean to false and make it optional for these methods.
- handleChangeForAllTrees
- promptChangeForAllTrees

Reply:
nothing in particular, other than to save some extra characters when calling the method 😋